joinus.html.haml

```diff
@@ -96,13 +98,16 @@ title: Join Us
                     %a{:href => "https://github.com/wildfly"}Source Code
                   %li
                     %i.icon-pencil
-                    %a{:href => "http://jira.jboss.org/browse/WFLY"}Issue tracker
+                    %a{:href => "http://jira.jboss.org/browse/WFLY"}WildFly Full Issue tracker
+                  %li
+                    %i.icon-pencil
+                    %a{:href => "http://jira.jboss.org/browse/WFCORE"}WildFly Core Issue tracker
                   %li
                     %i.icon-road
                     %a{:href => "https://issues.jboss.org/browse/WFLY#selectedTab=com.atlassian.jira.plugin.system.project%3Aroadmap-panel"}Road Map
                   %li
                     %i.icon-comments
-                    %a{:href => "irc://irc.freenode.net/#wildfly"} IRC 
+                    %a{:href => "https://www.hipchat.com/gW90m6pIs} HipChat
```

Comment:
just for future reference this is missing closing quote and as result fails build. 
Fixed in 48c8481a2257a6c7fd3daa9aa35d62391c17c78a